Security/verify_IPS_from_the_command_line.md

```diff
@@ -0,0 +1,158 @@
+{{{
+  "title": "Verify IPS Install from the Command Line",
+  "date": "09-15-2015",
+  "author": "Matthew Close",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": false
+}}}
+
+### Verify IPS Install from the Command Line
+
+#### Overview
+
+This tutorial will show how to trigger a CenturyLink Cloud IPS rule and view the notification using webhooks.
+
+#### Requirements
+
++ A CenturyLink Cloud host running the IPS service. To get started with IPS, please follow the instructions in [Getting Started with Intrusion Prevention System (IPS)](../Security/getting-started-with-ips.md)
++ ruby
++ sinatra
++ ngrok
++ curl
+
+**Note:** It is possible to complete this test without installing ruby, sinatra, and ngrok by using other HTTP request inspectors. For example, you could use [requestb.in](http://requestb.in). If you choose to use another inspector, you can skip to the second section. However, please be sure to use the URL provided by [requestb.in](http://requestb.in) instead of the ngrok URL.
+
+#### Instructions for setting up your webhook listener
```

Comment:
i'm not married to it.  just thinking from an ease of use view.  if you want to leave it in for advanced users perhaps just have the 2 options, put in a little table of contents to the kb and people can flip to which makes sense.  on some of my longer in depth kbs there isn't always 1 approach.